upf/upf_ipfilter.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright(c) 2018 Travelping GmbH.
+ * Copyright(c) 2021 Travelping GmbH.
```

Comment:
updating the copyright is also no needed and only bloats the diff

Reply:
Removed the updates in `upf/` from the patch